### ISSUE TITLE
String 'Wikipedia' not marked for translation

### ISSUE THREAD

**Issue by verdulo** [osmandapp/Osmand#3562]
If you click on properties of a POI, the field 'Wikipedia' is not translated - it has not been added to the Weblate.

**Comment by verdulo** [comment]
Another problem: why translations are not included in 'default' night versions?

**Comment by vshcherb** [comment]
They should be included do you have any example? Can you send a link to default version you used.

**Comment by verdulo** [comment]
I'm using nightly version downloaded from http://download.osmand.net/latest-night-build/OsmAnd-default.apk and later updated with menu 'Builds'=>'default'. Since the addition of depth contour plugin all translations are not working - system language is Polish, no matter which language I select in program, it's always displayed in English. Android version 6.0.

**Comment by vshcherb** [comment]
As I see the changes are still committed https://github.com/osmandapp/Osmand/blob/master/OsmAnd/res/values-pl/strings.xml and I don't see any errors in the builds. In the test build I do see these changes https://github.com/osmandapp/Osmand/commit/252680b8f0bdbd2144f888c1746d96d5b67649a3

**Comment by verdulo** [comment]
I'm not talking about new strings - the whole program is always in English, I've checked it an another device and the problem exists too.

**Comment by vshcherb** [comment]
Yes I see "default" apk is somehow 10MB less. I wiill clean the whole build and restart it.

**Comment by vshcherb** [comment]
The file supposed to be 60MB same as others http://download.osmand.net/latest-night-build/

**Comment by Zahnstocher** [comment]
freedev now has resConfig "en":
https://github.com/osmandapp/Osmand/commit/22ec0f69a8638b885f2224935b952b72d6011396
Probably this is the problem.

**Comment by verdulo** [comment]
I previously suggested (https://github.com/osmandapp/Osmand/issues/3008) that there's shouldn't be any string displayed, only the icon 'W'.
